Rakefile

```diff
@@ -8,3 +8,79 @@ Rake::TestTask.new do |t|
   t.test_files = FileList['test/**/*test.rb']
   t.verbose = true
 end
+
+def print(family, template, distro, major, minor, distro_prefix, release, validate, ipv4_subnet, ipv6_subnet)
+  renderer = TemplateRenderer.new(family, template)
+  code, stdout, stderr, contents = renderer.render(distro, major, minor, distro_prefix, release, validate, ipv4_subnet, ipv6_subnet)
+  puts contents
+  puts stdout
+  puts stderr
+  code
+end
+
+$LOAD_PATH << "test"
+require "test_helper"
+desc "Render a selected template"
+namespace :render do
+  namespace :redhat8 do
+    namespace :dhcp do
```

Comment:
Can we loop over this to reduce duplication? Make it a parameter to the task perhaps?

Reply:
Well it did not work for me, I think I cannot use namespaces in loops. Will convert it into a single rake task with parameters. Oh boy, rake and parameters - such combination.